getdess.sh

```diff
@@ -140,7 +156,13 @@ install_docker () {
           amzn) $pkg_man install -y libffi libffi-devel openssl-devel python3 python3-pip python3-devel gcc;;
           *) $pkg_man install -y python3 python3-pip;;
         esac;
-        pip3 install docker-compose;
+        # Setting up docker-compose in a virtualenv
+        python3 -m venv .venv;
+        source .venv/bin/activate;
+        compose_version=$(curl https://api.github.com/repos/docker/compose/releases/latest | jq .name -r)
```

Comment:
jq is not installed by default on all systems

Reply:
Python3 contains pyjq. Is this something we should pivot onto instead?

Reply:
Possibly. But I'm also wary about using a venv around docker-compose, so this entire approach needs to be reevaluated.